qr-code-component-main/main.css

```diff
@@ -0,0 +1,54 @@
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0;
+    min-height: 100vh;
+    font-family: Outfit;
+    background-color:  hsl(212, 45%, 89%);
+}
+.main_block {
```

Comment:
Formatting: add an empty line above for consistency. Check other places in this file for same problem